iwyu.cc

```diff
@@ -794,6 +794,11 @@ class BaseAstVisitor : public RecursiveASTVisitor<Derived> {
       return true;
 
     if (FunctionDecl* fn_decl = DynCastFrom(expr->getDecl())) {
+      if (!IsTemplatizedFunctionDecl(fn_decl) ||
```

Comment:
Ugh, the formatting here is pretty terrible... Could you trick it by introducing a temporary?